themes/nether-labels/README.md

```diff
@@ -0,0 +1,12 @@
+<!--
+SPDX-FileCopyrightText: Elfi Ox <https://github.com/elfi-ox>
+SPDX-License-Identifier: CC-BY-1.0
```

Comment:
I think [you should use 4.0](https://creativecommons.org/licenses/by/1.0/deed.de).

```suggestion
SPDX-License-Identifier: CC-BY-4.0
```